packages/botonic-react/src/webchat/hooks/use-scroll-to-bottom.ts

```diff
@@ -0,0 +1,25 @@
+import { getWebchatElement } from '../../util/dom'
+import { BotonicContainerId } from '../constants'
+
+export const useScrollToBottom = ({
+  host,
+  behavior = 'smooth',
+  timeout = 200,
+}) => {
+  const scrollToBottom = () => {
+    const webchatElement = getWebchatElement(host)
+    if (!webchatElement) return
+    const scrollableMessagesList = document.getElementById(
```

Comment:
investigate why you need to work with id instead of ref in this hook

Reply:
fixed in last commit